lang/en/typeshed/stdlib/microbit/microphone.pyi

```diff
@@ -53,10 +54,19 @@ def set_threshold(event: SoundEvent, value: int) -> None:
 
     Example: ``microphone.set_threshold(SoundEvent.LOUD, 250)``
 
-    A high threshold means the event will only trigger if the sound is very loud (>= 250 in the example).
+    The ``SoundEvent.LOUD`` event is triggered when the sound level crosses the 
+    threshold from "quiet" to "loud", and the ``SoundEvent.QUIET`` event is 
+    triggered when the sound level crosses from "loud" to "quiet".
+
+    If the ``SoundEvent.LOUD`` threshold is set lower than the 
+    ``SoundEvent.QUIET`` threshold, then the ``SoundEvent.QUIET`` threshold 
+    will decrease by one unit below the ``SoundEvent.LOUD`` threshold. If the 
+    ``SoundEvent.QUIET`` threshold is set higher than the ``SoundEvent.LOUD``
+    threshold, then the ``SoundEvent.LOUD`` threshold will increase by one unit 
+    above the ``SoundEvent.QUIET`` threshold.
 
     :param event: A sound event, such as ``SoundEvent.LOUD`` or ``SoundEvent.QUIET``.
-    :param value: The threshold level in the range 0-255.
+    :param value: The threshold level in the range 0-255. Values outside this range will be clamped.
```

Comment:
This is an update based on recently [merged micropython doc PR](https://github.com/bbcmicrobit/micropython/pull/804/files). 

https://github.com/microbit-foundation/micropython-microbit-stubs/pull/97/commits/f5ca4aaf4f75db225cb50cd2c1b90e7b7487d46e - I copied `set_threshold` doc from the PR
https://github.com/microbit-foundation/micropython-microbit-stubs/pull/97/commits/936b1723bb85541d66b82e3976a4477e1c7e1322 - my attempt in updating it

I have not updated the Python Editor beta PR typeshed files yet, because in previewing the python editor, the docs look quite verbose, so I am tempted to cut it down, but am not sure how  
<img width="414" alt="Screenshot 2024-04-12 at 17 17 21" src="https://github.com/microbit-foundation/micropython-microbit-stubs/assets/145345672/ab8eaf1c-372e-4e24-8c72-35f53b842fde">